client/src/providers/User.js

```diff
@@ -57,6 +59,9 @@ export const PROFILE = gql`
       id
       name
       description
+      image {
+        uri
```

Comment:
I wonder if it needs a sub property 'uri' could it not just be profileImageUri ?